### PR TITLE
multimc: 0.6.7 -> 0.6.11

### PR DESCRIPTION
**Pull request by anna328p**
###### Motivation for this change

This version of multimc allows installing Forge on Minecraft versions >= 1.13.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/multimc/default.nix [hunk 1]
```diff
@@ -1,10 +1,10 @@
-{ stdenv, mkDerivation, fetchFromGitHub, cmake, jdk, zlib, file, makeWrapper, xorg, libpulseaudio, qtbase }:
+{ stdenv, fetchFromGitHub, cmake, jdk, zlib, file, makeWrapper, xorg, libpulseaudio, qtbase }:
 
 let
   libpath = with xorg; stdenv.lib.makeLibraryPath [ libX11 libXext libXcursor libXrandr libXxf86vm libpulseaudio ];
-in mkDerivation rec {
+in stdenv.mkDerivation rec {
```
Comment (worldofpeace) [inline pkgs/games/multimc/default.nix:5]: please revert this change here that changes what deriver is used.
This package uses `libsForQt5.callPackage` which brings the `qt5` specific `mkDerivation` and libraries into scope.

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Forgot to request changes. See above comment.

**Comment by dkudriavtsev** [comment]
@worldofpeace Done

**Comment by worldofpeace** [comment]
20.03 backport 92d43c9279fb00fbfde4214b23240927b5928ec8
